traits/trait_types.py

```diff
@@ -1913,16 +1913,24 @@ class BaseEnum(TraitType):
         The enumeration of all legal values for the trait.  The expected
         signatures are either:
 
-        - a single list, enum.Enum or tuple.  The default value is the first
-          item in the collection.
+        - a collection.  The default value is the first item in the
```

Comment:
Thanks for the updates to the docstring; please could you also update the `Enum` docstring so that `BaseEnum` and `Enum` match?